pages/glossary/owner-element.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Owner element
+key: owner-element
+---
+
+
+If the node is referenced by an element through the `aria-owns` attribute, then the owner element is the first node in the DOM tree that references it through `aria-owns`.
```

Comment:
Should it be "the first node in the DOM tree that is not hidden using WAI-ARIA or CSS and that references it through `aria-owns`." ?

Reply:
@WilcoFiers, first node, would that be traversing up through the DOM from the referencing/owned element? Or down from the top of the DOM tree?

Reply:
Depending on the answer for this, Failed example 7 might have to be changed

Reply:
My interpretation of the Core Accessibility API mappings is that it is from the top of the DOM tree (which makes failed example 7 a fail). But I might be wrong since it is not explicit.

Reply:
@annethyme That's a good question. I would say that as long as the element is in the DOM, its "aria-owns" claim should persist, so that putting aria-hidden="true" on it means it and all its owned elements are considered hidden, and not cause its owned elements to change owner. I can easily see that being decided differently if this ever makes it to a standards group, but for now, my vote would be to keep what is here. It's also the simplest way to look at it.

Reply:
@WilcoFiers, added a note reflecting your view on hiding the owner element's consequence for owned elements.

Reply:
@WilcoFiers, can you agree to what Carlos also said on this: 
first node, would that be traversing up through the DOM from the referencing/owned element? Or down from the top of the DOM tree?

Reply:
Starting from the root node, yes.